Sources/SourceKit/DocumentManager.swift

```diff
@@ -135,6 +135,13 @@ public final class DocumentManager {
       return document.latestSnapshot
     }
   }
+
+  /// All currently opened documents.
+  public func openDocuments() -> [URL] {
```

Comment:
if we need this, I would make it:
```swift
public var openDocuments: [URL] { ... }
```

and mention in the doc comment that the order is arbitrary and may change.

Reply:
Swapped to Set